assets/src/edit-story/components/library/panes/text/textSets/raw/editorial.json

```diff
@@ -66,7 +66,24 @@
               [1, 900]
             ],
             "fallbacks": ["Helvetica Neue", "Helvetica", "sans-serif"],
-            "service": "fonts.google.com"
+            "service": "fonts.google.com",
```

Comment:
Should the version number increase in this fixture, as this is kind of a manual migration to the new data version?

Reply:
With the current logic, yes. At this moment it's still `24` though.